users/templates/users/profile_settings.html

```diff
@@ -1,79 +1,96 @@
+{% load static %}
+{% block content %}
 <div class="container mt-4">
     {% include 'users/settings_navbar.html' %}
     <div class="row justify-content-center">
         <div class="col-12 col-md-8">
             <div class="d-flex justify-content-between"></div>
             <div class="bg-dark p-4">
                 <h2 class="mt-3 mb-3">Customize Profile</h2>
-                <form>
-                    <div>
-                        <div class="form-group mb-4" id="profile-information">
-                            <h3 class="mt-4 mb-4 text-secondary fs-6 border-bottom border-secondary">Profile Information</h3>
-                            <label for="displayName" class="fw-bolder">Display Name (optional)</label>
-                            <p class="form-text text-light-emphasis">Set a display name. This does not change your username.</p>
-                            <input type="text" class="form-control text-red" id="text-field" placeholder="Display name (optional)" maxlength="30" data-max-chars="30" />
-                        </div>
-                        <div class="form-group mb-4">
-                            <label for="about" class="fw-bolder">About (optional)</label>
-                            <p class="form-text text-light-emphasis">A brief description of yourself shown on your profile.</p>
-                            <textarea class="form-control" id="about" rows="4" maxlength="200" placeholder="About (optional)"></textarea>
-                        </div>
-                        <div class="form-group mb-4">
-                            <h4 class="fw-bolder fs-6">Social Links (5 max)</h4>
-                            <p>People who visit your profile will see your social links.</p>
-                            <button type="button" class="btn btn-secondary">Add Social Link</button>
+                <form method="POST" enctype="multipart/form-data">
+                    {% csrf_token %}
+
+                    <!-- Display Name Field -->
+                    <div class="form-group mb-4" id="profile-information">
+                        <h3 class="mt-4 mb-4 text-secondary fs-6 border-bottom border-secondary">Profile Information</h3>
+                        <label for="nickname" class="fw-bolder">Nickname (optional)</label>
+                        <input type="text" class="form-control" id="nickname" name="nickname" maxlength="150" value="{{ user_form.nickname.value }}">
+                        <p class="form-text text-light-emphasis">Set a nickname. This does not change your username.</p>
+                                {% if user_form.nickname.errors %}
+                                    <div class="text-danger">
+                                    {% for error in user_form.nickname.errors %}
+                                        <p>{{ error }}</p>
+                                    {% endfor %}
+                                    </div>
+                                {% endif %}
```

Comment:
```suggestion
{{ user_form.nickname}}
```

Use this:                         <p class="form-text text-light-emphasis">Set a nickname. This does not change your username.</p> in help text (w/o paragraph)

Reply:
Done